src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<blockquote>It was decided to put the <strong>MTOW</strong> computation in its own model main function to highlight its relevance, both from the aircraft sizing point of view and because of its key role in the sizing process. </blockquote><p>Not sure about the need to put this sentence. In any case, for less verbosity, I'd rather begin with "the MTOW computation is put in its own...".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Deleted entirely in dfe233f7